apps/api/src/lib/utils.ts

```diff
@@ -64,11 +71,14 @@ export function normalizeCourse(course: PrismaCourse): Course {
   return {
     ...course,
     courseLevel,
-    instructorHistory: course.instructorHistory as unknown as string[],
+    instructorHistory: course.instructorHistory,
+    instructors: course.instructors as unknown as InstructorPreview[],
     prerequisiteTree: course.prerequisiteTree as unknown as PrerequisiteTree,
```

Comment:
It seems to work for me without the first assertion to unknown.

```suggestion
    instructors: course.instructors as InstructorPreview[],
    prerequisiteTree: course.prerequisiteTree as PrerequisiteTree,
```